packages/eslint-config-hypothesis/index.js

```diff
@@ -2,7 +2,10 @@
 
 module.exports = {
   env: {
-    es6: true,
+    // Specify global variables and ES language version.
+    //
+    // See https://eslint.org/docs/user-guide/configuring#specifying-environments
+    es2020: true,
```

Comment:
New versions of ESLint support using either a year or a version number to specify ES versions. The year seems more familiar.